pages/organization/[organizationId]/index.js

```diff
@@ -44,17 +42,36 @@ const Dashboard = () => {
 
   const [org, setOrg] = useState({})
 
-  function resetLoaders () {
-    setTopLevelPackagesLoading(true)
+  function resetDonationLoaders () {
     setDonationLoading(true)
-    setOrgDepCountLoading(true)
+    setTotalContributionsAmountLoading(true)
   }
 
-  async function fetchData () {
-    if (!router.query || !router.query.organizationId) return 
+  async function fetchAllData () {
+    await fetchOssUsageData()
+    await fetchDonationData()
```

Comment:
Since these calls don't depend on each other, should get a good speed up by running them in parallel with `Promise.all`.

Reply:
amazing point, thanks